### PR TITLE
Add recipe sherpas

### PR DESCRIPTION
**Pull request by blinard-BIOINFO**
A new recipe for SHERPAS, an alignment-free genome recombination detection tool exploiting the idea of phylo-kmers to accelerate recombination detection by several orders of magnitude while keeping comparable accuracy.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by blinard-BIOINFO** [comment]
@BiocondaBot please add label

File: recipes/sherpas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{% set name = "sherpas" %}
+{% set version = "1.0.0" %}
+{% set sha = "215729efbfe969b51626716e470e4570ec1c32a383e1646401f5ce708e8446fd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/phylo42/{{ name|lower }}.git
```
Comment (npavlovikj) [inline recipes/sherpas/meta.yaml:10]: Is there a reason why `git_url` is used here? We usually recommend using the release, e.g. `https://github.com/phylo42/sherpas/archive/1.0.0.tar.gz`.

Comment (blinard-BIOINFO) [reply recipes/sherpas/meta.yaml:10]: Dear Natasha,
The main problem is that "releases" archives built via github do not include the GIT submodules configured in the repository. And from what I read n forums, github is not going to change this in the near future.

Until github allows to built releases including submodules, the `git_url` seems a much better option for maintenance, of recipes involving many submodules.

As a workaround I will use update the recipe with the manually built archive: `https://github.com/phylo42/sherpas/releases/download/1.0.0/1.0.0.tar.gz`

The github url is then different (release/download/, not /archive/), is this ok with future auto_bumps ?

**Comment by blinard-BIOINFO** [comment]
@BiocondaBot please update

**Comment by npavlovikj** [comment]
@BiocondaBot please fetch artifacts

**Pull request merged**